### PR TITLE
Convert node handler to sidecar impl

### LINKED ISSUE #31: [sidecar] update the Node handler to be an "invoker"

### PR DESCRIPTION
**Pull request by ericbottard**
Fixes #31

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -64,9 +64,10 @@ kubectl create -f samples/echo-function.yaml
 
 == Watch Echo Function Sidecar Log
 
+In another terminal
 ```
-# In another terminal
-kubectl logs -f [echo-pod-name] -c sidecar
+#kubectl logs -f -c sidecar [echo-pod-name]
```
Comment (markfisher) [inline README.adoc]: did you intentionally leave this commented out line here?

File: function-invokers/node-function-invoker/src/main/docker/image/server.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+const PORT = 8080;
+
+const app = express();
+
+app.use('/', bodyParser.text());
+
+var fn = require(process.env.FUNCTION_URI);
+
+app.post('/', function (req, res) {
+	var resultx = fn(req.body);
```
Comment (markfisher) [inline function-invokers/node-function-invoker/src/main/docker/image/server.js]: extra indent here

File: README.adoc (hunk 1 above)
Comment (ericbottard) [reply README.adoc]: yes, but can remove

File: function-invokers/node-function-invoker/src/main/docker/image/server.js (hunk 2 above)
Comment (ericbottard) [reply function-invokers/node-function-invoker/src/main/docker/image/server.js]: Fixed

**Comment by markfisher** [comment]
@ericbottard can you please rebase this on the current master branch?

**Comment by ericbottard** [comment]
Done

File: build-model [hunk 3]
```diff
@@ -3,6 +3,6 @@ set -e -v
 
 # regenerate the fabric8 java API including k8s custom resources
 pushd $(go env GOPATH)/src/github.com/fabric8io/kubernetes-model/
-git checkout sk8s
+git checkout sk8s-sidecar
```
Comment (markfisher) [inline build-model]: @ericbottard can you please merge `sk8s-sidecar` into `sk8s` and then submit a PR (separate from this one which I'm about to merge) to update this one line?

Comment (ericbottard) [reply build-model]: Sure

**Comment by markfisher** [comment]
rebased, squashed, and merged as https://github.com/markfisher/sk8s/commit/f0ec7a93b09d46c7de921e19e4b82b1762795284

**Pull request closed without merging**